MANUAL.md

```diff
@@ -140,6 +140,11 @@ Available plugin options:
   with our option (ts.exclude_options). Set this option if you are certain you do not want
   to include any options at all.
 
+- "keep_enum_prefix"
+  By default, if all enum values share a prefix that corresponds with the enum's name,
+  the prefix is dropped from the value names. Set this option if you want to disable this behavior
+  and keep the prefix intact.
```

Comment:
Do we keep the prefix intact, or the enum value name? Suggest to keep it simple:

```suggestion
- "keep_enum_prefix"
  By default, if all enum values share a prefix that corresponds with the enum's name,
  the prefix is dropped from the value names. Set this option to disable this behavior.
```